Dockerfile.sdk

```diff
@@ -126,8 +129,7 @@ RUN pip3 install --upgrade numpy pillow && \
     xargs pip3 install --upgrade
 
 # Install DCGM
-RUN wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/datacenter-gpu-manager_${DCGM_VERSION}_amd64.deb && \
-    dpkg -i datacenter-gpu-manager_${DCGM_VERSION}_amd64.deb
+RUN apt-get update && apt-get install -y datacenter-gpu-manager=1:${DCGM_VERSION}
```

Comment:
can ignore these changes. These are for building the container with the recent cuda changes.

Reply:
@pragyaarora